src/main/java/matchmaker/backend/controllers/DepartmentController.java

```diff
@@ -51,6 +51,10 @@ public ResponseEntity<Department> createDepartment(
       // user is not in the same company
       return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
     }
+    if (targetDepartmentAdmin.role.isDepartmentAdmin) {
+      // target admin is already a department admin
+      return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
```

Comment:
Er wordt heel veel BAD_REQUEST gegeven hier, zijn er geen andere meer sprekende statussen?

Reply:
Hmm fair point. Het ding is, als de target admin:  
1. niet bestaat 
2. niet in een bedrijf zit 
3. al admin is 
4. in een ander bedrijf zit

gaat allemaal over de target user. In de frontend krijg je een gefilterde keuze obv mensen die hieraan voldoen